### ISSUE TITLE
Update docs/logic with `POST /api/v3/posts` endpoint

### ISSUE THREAD

**Issue by weerd** [DoSomething/rogue#599]
_DOCUMENTATION_
-------------------
#### Affected Pages
[Create a Post](https://github.com/DoSomething/rogue/blob/master/documentation/endpoints/v3/posts.md#create-a-post)

#### Explanation
```
campaign_run_id: (int) optional. The drupal campaign run node id of the campaign that the user's post is associated with.
```
This item specifying the `campaign_run_id` as optional needs to be updated to indicate that it is required if the `campaign_id` is a Drupal ID. 

One thing to note is that the `campaign_id` value can be either an integer or a string... and the Drupal ID can apparently be either (`362` or `"362"`), but I doubt that's the case for the `campaign_run_id`. So I think it would make sense to also update the docs to specify that the `campaign_id` should be an `int` if a Drupal ID or a `string` if a Contentful ID. Thoughts?

**Comment by DFurnes** [comment]
I believe we're just checking if it's numeric or not (`/[0-9]+/`), since even if it's provided by the client as an integer it'd still be casted & stored as a string.

**Comment by weerd** [comment]
yeah, but then there's inconsistency cause in the scenario of the Drupal ID I could pass the `campaign_id` as a string but for the `campaign_run_id` it _needs_ to be an integer. Maybe just update the `campaign_run_id` to accept both as well?

**Comment by DFurnes** [comment]
Same deal there I believe, Laravel's `integer` validator will just check if the provided value can be used as an integer (whether it's provided as a string or integer in the JSON body).

**Comment by weerd** [comment]
Okie dokes. Then just a docs update to indicate as such for future ref that passing a string to the `campaign_run_id` is cool too :) 🎉
